subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -500,7 +518,26 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 		{"channel", required_argument, 0, 'c'},
 		{"timeout", required_argument, 0, 't'},
 		{"anon-id", required_argument, 0, 'a'},
-		{"key-passwd", required_argument, 0, 'K'},
+		{"key1-pwd", required_argument, 0, 'K'},
```

Comment:
Add long opt to support multi key passwords , also for user identities\passwords.